### PR TITLE
Added Github_Unfollowers Script

### LINKED ISSUE #732: Find Unfollowers on Github
## Description 

There are a lot of people who follow people and unfollow them as well, later. While we don't exactly know, under what circumstances is the `Unfollow` button clicked, but we for sure, don't know who unfollowed us. 

Build a Script/CLI that can track the Usernames who unfollowed you recently on Github. 

## Languages 

- Python
- Javascript
- Golang
- BASH

### PR DESCRIPTION
**Pull request by vismitap**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
This is a script to track recent unfollowers on github

## Type of change

_Please delete options that are not relevant._

- [x ] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

Output:
output

### CODE REVIEW

**Comment by vismitap** [comment]
Hey! Forgot to edit the main README.md and add my name. Doing one commit for that!

File: JavaScript/Github_unfollowers/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/.gitignore:1]: Please remove the `.gitignore`. We have a global `.gitignore` to cover everything up :)

File: JavaScript/Github_unfollowers/Github_Unfollowers.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// Script to track recent unfollowers on github using github api
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/Github_Unfollowers.js:1]: Replace `Script to track recent unfollowers on github using github api` to `Script to track recent people who unfollowed you on Github using the Github API`

File: JavaScript/Github_unfollowers/Github_Unfollowers.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    if (current.length == 0) {
+      console.log("Run for the first time.");
+    } else {
+      const unfollowers = current.filter((e) => !set.has(e));
+      console.table(unfollowers);
+    }
+    fs.writeFileSync("old.json", JSON.stringify(Array.from(set)));
+  })
+  .catch((err) => console.log(err));
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/Github_Unfollowers.js:38]: Add a new line at the EOF

File: JavaScript/Github_unfollowers/Github_Unfollowers.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    let i = 0;
+    let set = new Set();
+    
+    // find the followers username on github from the returned json by github API
+    for (i in out) {
+      set.add(out[i]["login"]);
+    }
+
+    if (current.length == 0) {
+      console.log("Run for the first time.");
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/Github_Unfollowers.js:31]: Use a more friendly terminal statement like: `Analyzed and Run for the first time`

File: JavaScript/Github_unfollowers/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Github_Unfollowers
+This is a script to track recent unfollowers on github.
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/README.md:2]: Replace this with `Github Unfollowers is a Script to track the recent accounts who unfollowed you on Github`

File: JavaScript/Github_unfollowers/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Github_Unfollowers
+This is a script to track recent unfollowers on github. 
+* This script uses the github API to fetch data of followers.
+* The data is then stored in old.json file and compared with new data and hence the unfollowers can be found!
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/README.md:4]: Use backticks for `old.json` file. If possible you can replace this with a more friendly or understanding filename, though that's entirely optional

File: JavaScript/Github_unfollowers/README.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+# Setup instructions
+* cd to Rotten-Scripts\JavaScript\Github_unfollowers.
+* Run `npm install` to install all dependencies.
+* Check whether there is [] in old.json in the beginning.
+* Run `node Github_unfollowers.js` and unleash the magic ✨✨.
+
+# Output
+![Run First Time](https://i.imgur.com/AjsMeNy.png)
+![Run Second Time](blob:https://imgur.com/d45a7353-de3b-420e-a459-16670c0f62ca)
```
Comment (HarshCasper) [inline JavaScript/Github_unfollowers/README.md:14]: This doesn't work. Can you optimize this?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Exceptional work. Requested for some minor changes. 

Thanks!

**Comment by vismitap** [comment]
Sure! I will do the changes and revert back 🤩👍

**Comment by vismitap** [comment]
Hey! Messed up something really badly.  I will make a fresh PR with all those changes. Sorry for the inconvenience.😓

**Pull request closed without merging**